geth/geth-entrypoint

```diff
@@ -53,6 +52,14 @@ if [ "${HOST_IP:+x}" = x ]; then
 	ADDITIONAL_ARGS="$ADDITIONAL_ARGS --nat=extip:$HOST_IP"
 fi
 
+if [ "$OP_NODE_NETWORK" = lisk-mainnet ]; then
+    ADDITIONAL_ARGS="--override.holocene=1736445601"
+elif [ "$OP_NODE_NETWORK" = lisk-sepolia ]; then
+    ADDITIONAL_ARGS="--override.holocene=1732633200"
+else
+    ADDITIONAL_ARGS=""
+fi
+
```

Comment:
Can we please move the timestamps as environment variables to the respective .env file and use them in the script instead?